importers/src/bible/sword/src/importer.ts

```diff
@@ -29,8 +30,10 @@ export class SwordImporter extends BibleEngineImporter {
                         title: swordModule.config.description,
                         language: swordModule.config.language,
                         copyrightShort: swordModule.config.shortCopyright,
+                        isPlaintext: !xml.includes(`</${OsisXmlNodeName.PARAGRAPH}>`),
```

Comment:
there are different ways to start paragraphs, don't know if sword modules are consistent here? and i also don't know currently what kind of paragraphs the osis importer supports. at least this should catch all the ways that the osis importer does.

Reply:
Sword modules have inconsistent paragraph usage, so I don't include their tags in the `.includes` here

Reply:
wouldn't this then flag this import as plaintext only if a source file does not include `OsisXmlNodeName.PARAGRAPH` (but a different type of paragraph)?